src/frontend/Komponenter/Behandling/Inngangsvilkår/Medlemskap/Utenlandsopphold.tsx

```diff
@@ -1,38 +1,46 @@
 import * as React from 'react';
 import { FC } from 'react';
-import { formaterNullableIsoDato } from '../../../../App/utils/formatter';
-import TabellVisning from '../../Tabell/TabellVisning';
-import { VilkårInfoIkon } from '../../Vilkårpanel/VilkårInformasjonKomponenter';
 import { IUtenlandsopphold } from './typer';
+import Informasjonsrad from '../../Vilkårpanel/Informasjonsrad';
+import { BodyShort } from '@navikt/ds-react';
+import styled from 'styled-components';
+import { VilkårInfoIkon } from '../../Vilkårpanel/VilkårInformasjonKomponenter';
 
 interface Props {
     utenlandsopphold: IUtenlandsopphold[];
 }
+const PeriodeStyling = styled(BodyShort)`
+    font-weight: bold;
+    font-size: var(--a-font-size-medium);
+    text-decoration: underline;
+    margin-left: 29px;
```

Comment:
Synes PRen ser bra ut, men stusset litt på 29px margin her. Denne verdien fremstår som veldig magisk for meg, og må settes slik det er nå siden du tar i bruk Informasjonsrad som har tre kolonner per rad, når du egentlig bare har bruk for to kolonner. 

Jeg ville nok bare laget min egen grid i denne komponenten og tatt den i bruk i stedet for å ta i bruk InformasjonsradContainer. Gjerne spør hvis dette ble uklart :)